.gitignore

```diff
@@ -9,7 +9,8 @@
 /config/database.yml
 
 /app/graphql/*/.keep
-.env.development.local
+.env.*.*
+.env.*
```

Comment:
It's better to add .env.test and .env.development to the repo. This files should contain list of all keys used for configuration. "Safe" (i.e. not secret) and default/dummy values could be stored here. Each developer should create .env.{env}.local files to override values, like DB password and user. 

So, I presume, this files may not be added to gitignore. Their purpose to provide example configuration and default values.